### PR TITLE
Add Healthcheck to Multiroot

### PR DESCRIPTION
**Pull request by efabens**
- Adding healthcheck to multiroot server makes k8s probes easier to use

### CODE REVIEW

**Review by galxy25 — APPROVED** [review]

**Review by willmichael — APPROVED** [review]
is 204 required for something?

We also have this https://github.com/tozny/utils-go/blob/master/server/handlers.go

File: cmd/multirootca/ca.go [hunk 1]
```diff
@@ -92,6 +92,9 @@ func main() {
 	}
 
 	http.HandleFunc("/api/v1/cfssl/authsign", dispatchRequest)
+	http.HandleFunc("/api/v1/cfssl/healthcheck", func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(204)
```
Comment (galxy25) [inline cmd/multirootca/ca.go:96]: if only we had a [200 level version of this](https://http.cat/418) for "I'm a mint tea pot"

**Review by galxy25 — APPROVED** [review]

**Comment by efabens** [comment]
204 because there is no content returned and I didn't pull in utils-go because it didn't seem neccesary for a simple function that will pretty much only be used by an automated tool.

File: cmd/multirootca/ca.go (hunk 1 above)
Comment (efabens) [reply cmd/multirootca/ca.go:96]: yeah k8s requires 200-399 for this. I love me a teapot

**Pull request merged**